.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,32 @@
+# JIRA-123 One Sentence Overview
```

Comment:
I don't like this line. It is (or it should be) already in the PR title just above. What's the point of duplicating it? It will be copy pasted anyway.